src/lib/eco-router/api.ts

```diff
@@ -122,6 +123,27 @@ export async function getExactIn(
       })
   })
 
+  const velodromeTrade = new Promise<any | undefined>(resolve => {
+    if (!RoutablePlatform.VELODROME.supportsChain(chainId)) {
+      return resolve(undefined)
+    }
+
+    VelodromeTrade.getQuote({
+      quoteCurrency: currencyOut,
+      amount: currencyAmountIn,
+      maximumSlippage,
+      recipient: receiver,
+      tradeType: TradeType.EXACT_INPUT,
+    })
+      .then(res => resolve(res ? res : undefined))
+      .catch(error => {
+        console.log('velError')
+        console.error(error)
+        errors.push(error)
+        resolve(undefined)
+      })
+  })
+
```

Comment:
This will go to the SDK, right?

Reply:
yeah it doesn't include the changes for the other pr once we merge it will change